pysisyphus/calculators/DFTD3.py

```diff
@@ -0,0 +1,64 @@
+from pysisyphus.calculators.Calculator import Calculator
+import re
+import numpy as np
+import os.path
+
+class DFTD3(Calculator):
+    
+    conf_key = "dftd3"
+
+    def __init__(self, geom, functional, bjdamping=False, **kwargs):
+        super(DFTD3, self).__init__(**kwargs)
+        self.atoms = geom.atoms
+        self.functional = functional.lower()
+        self.bjdamping = bjdamping
+
+        self.parser_funcs = {
+            "energy": self.parse_energy,
+            "grad": self.parse_gradient,
+        }
+
+    def parse_energy(self, path):
+        with open(path / self.out_fn) as handle:
+            text = handle.read()
+        mobj = re.search(r"Edisp /kcal,au:\s+([\d\-\.]+)\s+([\d\-\.]+)", text)
+        results={
+            "energy": float(mobj.group(2)),
+        }
+        return results
+
+    def parse_gradient(self, path):
+        grad = np.loadtxt(os.path.join(path,"dftd3_gradient"))
```

Comment:
The `path` is actually a pathlib.Path and overloads the `/` operator to allow path concatenation.

So this line should be
`grad = np.loadtxt(path / "dftd3_gradient")`